invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import {
+  Grid,
+  Segment,
+  Container,
+  Header,
+  Button,
+  Icon,
+} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { MetadataTable } from '../../../components/MetadataTable';
+
+export default class DocumentMetadata extends Component {
+  openEditor(url) {
+    window.open(`/editor?url=${url}`, url);
+  }
+
+  render() {
+    const data = {
+      ...this.props.documentDetails,
+      ...this.props.documentDetails.metadata,
+    };
+
+    const documentUrl = this.props.documentDetails.links.self;
+
+    const rows = [
+      { name: 'Created', value: data.created },
```

Comment:
If you serialize, you can format the dates here

Reply:
I've made it consistent with the ItemMetadata and LoanMetadata so I will change all of them to fix that